src/ScaffoldPackageCommand.php

```diff
@@ -522,8 +531,12 @@ public function package_tests( $args, $assoc_args ) {
 			$package_dir = rtrim( $package_dir, '/' );
 		}
 
-		if ( ! is_dir( $package_dir ) || ! file_exists( $package_dir . '/composer.json' ) ) {
-			WP_CLI::error( "Invalid package directory. composer.json file must be present." );
+		if ( ! is_dir( $package_dir ) ) {
+			WP_CLI::error( 'Directory does not exist.' );
+		}
+
+		if ( ! file_exists( $package_dir . '/composer.json' ) ) {
+			WP_CLI::error( 'Invalid package directory. composer.json file must be present.' );
 		}
```

Comment:
Given the repetition of code, what do you think about creating `self::check_package_dir( $package_dir )`, and abstracting these error checks to it?